solidstart-tanstackquery-tailwind-modules/README.md

```diff
@@ -33,6 +33,11 @@ SolidStart is a meta framework for SolidJS that is built on top of the React API
 - [ESLint](https://eslint.org/) - Code linting
 - [Prettier](https://prettier.io/) - Code formatting
 
+#### Note*
+We encountered an issue using Storybook in this showcase involving routes, you can follow the discord discussion [here](https://discord.com/channels/722131463138705510/1072093687892426843)  
```

Comment:
I think it makes sense to give a brief summary of what the issue is before providing a link to follow the discussion.

Reply:
Ok